core/src/com/unciv/ui/overviewscreen/GlobalPoliticsOverviewTable.kt

```diff
@@ -174,7 +174,10 @@ class GlobalPoliticsOverviewTable (
         // wars
         for (otherCiv in civ.getKnownCivs()) {
             if(civ.isAtWarWith(otherCiv)) {
-                val warText = "At war with [${otherCiv.civName}]".toLabel()
+                val warText = if (viewingPlayer.knows(otherCiv))
```

Comment:
Instead of these,how abou a function 'getCivName', that returns the name if known and "Unknown civilization" if unknown
You have 3 usages in this file already

Reply:
Done :) I've also tested the translations

![image](https://user-images.githubusercontent.com/21990802/209352656-548d61eb-ae84-4eb8-876e-a93abbcdc458.png)